test/system/roles/public_health/dashboard/dashboard_verifier.rb

```diff
@@ -94,6 +94,7 @@ def verify_patient_info(patient, workflow, tab)
     verify_patient_field('assigned user', patient[:assigned_user]) unless %i[transferred_in transferred_out].include?(tab)
     verify_patient_field('state/local id', patient[:user_defined_id_statelocal])
     verify_patient_field('date of birth', patient[:date_of_birth].strftime('%m/%d/%Y'))
+    verify_patient_field('phone number', patient[:primary_telephone])
```

Comment:
This does not work. You have to format the phone number for the comparison

Reply:
Please use `PublicHealthDashboardTest` to verify

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1164/commits/8a6ad07e6034bf51cbba1485bf76d2d0c0fb3190